### DISCUSSION TITLE
posts/2024/05/me-resonance-world/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#62] [General]
# posts/2024/05/me-resonance-world/

https://guozheng.rbind.io/posts/2024/05/me-resonance-world/

**Comment by earfanfan** [comment]
吐个槽，由于咱俩博客主题相似，每次过来逛都有种看自己博客的错觉，哈哈。

这篇文章其实我老早就想留言，不过一直没大看懂，但我今天决定还是写下一些看法，尽管可能偏离了你写的主题。

其一，我觉得自我怀疑、自我否定是一种认识自己的过程，当然啦，学习本身就是一个不断面对质疑和挫败的过程。所以说每当我们得到一时一刻的结论，过后又再次推翻也是可以的，思考问题没能得出结论也是可以的。

其二，回应或者共鸣在我看来似乎代表着一种“感觉”，这里借用一下阿德勒的“共同体感觉”，感受到回应或共鸣会让我们感觉到自己的存在，感受到自己与他者共同存在，或者感受到自己与世界共同存在。

我认可每个人都有“认可欲求”，大家都会希望得到回应、认可、共鸣。我也认同你在文中提到共鸣有许多种类。可随着我的年纪增长，我渐渐发现，尽管我确实需要环境给予我什么，但同时我也总是想要避免受到太多环境的影响，我想要确立一个边界，在这个边界内只想做自己想做的事，只有自己认可自己也没关系。如果做一些形象的比喻的话，感觉以前心里好像有很多洞，所以需要回应和共鸣带来的认可去填充，或者说作为动力来源，不知道什么时候开始，可能是当我学会无条件爱别人以后，动力来源变丰富了，感觉自己就可以填满自己的心……

**Reply by residualsun1** [reply]
> 吐个槽，由于咱俩博客主题相似，每次过来逛都有种看自己博客的错觉，哈哈。

虽然知道吐槽是你的习惯，但这条莫名戳我笑点。

第一点看法，我觉得我逐渐在接受它，而且也在让自己习惯这个过程。我过去在学习某个知识时，会在表达它的过程中感到愉悦（无论是言说还是写作），但看到更专业的表达时便为自己的表达感到难为情。比如，你提到的就读于人口学专业的博主谈及的对于田野的看法，我也是认同的，但在认同的同时，我心里也多少感到”相形见绌“；最近，我又看了春花写的一些关于人类学学习的文章，虽然我也知道自己和别人在阅读量上有很多差距，但在为他的文字折服时，想到自己的文字又觉得稚嫩许多，又难为情了。但总不能因此就停下来了，该学还得学哈哈，学着学着，就会慢慢直面这些感受了。

第二点其实也是我想在文中表达的。

虽然无法真切知晓你所描述的感受，但我觉得听起来似乎是一个丰盈的状态。坦诚来说，我现在应该便仍然是心中有着许多需要回应和共鸣的孔洞的状态，但我并不十分抗拒，觉得是正常的。另外，我觉得”爱别人“是可以做到你说的”动力来源变丰富“，以及自己填满自己的内心，因为按理想的状态来看，爱别人应当是从主动爱他人这个过程，即向外馈赠的这个过程中获取到了滋养，而非通过索取他人来得到滋养（关于这一点，弗洛姆好像在《爱的艺术》中表达过这个意思，例如恋人之间，有人是因为寂寞、想要被爱，因而恋爱；但也有人只是因为”想爱“（主动）才恋爱。）不过，要无条件爱别人应该是一种需要训练的能力。

**Reply by earfanfan** [reply]
其实我的心也仍然是处处是孔洞的状态，只是其中一部分能被自己填满而已。我说的无条件的爱并不一定只是爱别人，爱狗也是，爱自己也是。当然其实我感觉还包括一种情况，无条件的爱也不一定非得是爱具体的人，也可以是爱世人爱世界这种，如果我愿意爱世人，其实等同于我愿意努力保护好自己的赤子之心，愿意一次次受伤以后仍然努力做一个真诚友善的人（PS这个难度超级超级超级大，但我仍然在挑战这种高难度）。

悄悄告诉你，在进入社会的这许多年里，我的心真的是被打穿了许多孔洞。再悄咪咪地说（PS小花应该不会摸到你这来吧），我后来知道去年小花突然变得无论如何不肯理我的理由了，真是打死我都万万想不到，竟然是她觉得在意见不合时我不耐烦地吼了她一句，她觉得这是她最难接受的。哈，我知道以后真是既震惊又寒心。像这种人际关系上受伤还有许多，有些真的把我戳得贼疼，前几个月戳的，现在想起来都觉得很疼很疼。呐，还有很多社会畸形现象给我戳的疼，比如30岁以后找不到工作，因为没有结婚生育就被拒绝……嘿，你看到这里是不是觉得超可怕的，会不会觉得原来进入社会以后还会被重锤，心会被戳出无数淌血的血洞洞。不得不说，哎哟喂，真的超级疼的说。

嗷，对了，被戳疼了，心里还会积攒很多戾气，所以我总是在不断地努力尽量去化解那些世界重伤我带来的戾气，不过呢，以前受了重伤自己没能力治愈的时候，会变得随便遇到什么人都当救命稻草一样抓住，现在被锤炼了那么多年，开始学习分辨清楚哪些是值得真心对待的人，哪些是不值得真心对待的人，即便是我依然需要很多回应和共鸣，但也明白好像与我能有回应和共鸣的人并不是随便什么人都行。

**Reply by residualsun1** [reply]
我想我现在能比刚收到你的回复时稍微更能理解一点点你说的“无条件的爱”了。相比之下，我感觉我之前想的似乎太理想化了一些。理论看起来是美好的，“主动地无条件爱别人”这样的话语对于现在的我来说是有鼓动性的，但在现实生活中，不论是让我付诸实践，还是让我去寻找愿意这样对待我的人，都很难很难。我觉得我还是经历得太少，和你谈论的时候隔着一些距离，最近发生的一些小事就让我想起你说到的“填满一部分的自己”、“爱自己”和“爱狗”、“爱世人”、“爱世界”。

> （PS小花应该不会摸到你这来吧）

（不用担心！我的文章都比较冗长，估计小花摸过来了也不会像你这样有耐心看下来哈哈😎）
我现在临近毕业，越发对进入社会感到害怕，前几天我的一位表哥突然给我发了自己的微信书单，和我说可以看看，毕竟还是要进入“丛林社会”。我知晓他对我的关心，而且所言也不虚，但看到的时候心里也着实黯然，难过，带着隐晦的害怕。关于你和小花的事情，我能感觉出你对情感的重视，以及一种仍然对天真的信任，如同你说的，愿意让自己还爱这个世界。因此才会有这样的反应。如果是一个从不愿意和别人交心的人，或者带着冷酷和算计的心思和他人打交道的人，又如何会在意别人为什么不理会自己呢？想到这些的时候，我反而觉得你和我当下假想的社会有一些不一样，在我想象的社会里，虽然抽象，但就和我表哥说的是丛林那般，我与他者之间基本没有感情可言，相互戒备才是常态。或许是年龄缘故，我其实还对“三十岁找不到工作”等没有太多想象，现在更担心的还是模糊的叙事：由于没有过硬的技术，只能在就业市场不景气的今天如无根浮萍漂泊，或是被各种工作单位当牛马驱使。我将眼睁睁看着大学四年读过的书籍、去过的地方给我的感受所建构出来的一种“还有更多可能”的崩溃瓦解，我将深刻意识到自己此后都要和生存以及看不见的期待周旋。还是前两天，我的母亲给我发来了一份她所在单位的招聘信息（在我们老家），我看到以后竟然感到莫名的愤怒、恐惧和悲伤的交织。我没有任何理由看不起母亲所在的单位，我尽力地去梳理自己感情的出处，里面一定有着孔乙己的长衫，有我发现自己读了这所谓大学，最后却要去从事一份一眼便似乎可以看见尽头，日复一日的工作，我的真正的灵魂将在大学毕业那一刻死亡；也有我感觉我母亲希望我回到她和父亲身边的意愿，一如我哥哥曾经满含热血要在外地做一番事业，最后还是灰溜溜回到家里，在父母的帮助下从事了一份工作，就连现在的恋爱对象也是母亲托人介绍来的。我又想起，在我过去还很敢说起一些理想化的话语时，我哥哥就喜欢嘲讽我，告诉我现实会教训我。我深深记住这些，或许正因如此，看到他的经历，更让我感到一阵悲伤，我渴求自己不要，或者说我不允许自己在毕业后就返回家乡，回到父母身边。我认为我还应该继续成长，我还要寻找一些可能性，我想远离他们。但我又知道，现在这些情感很大程度建立在父母无条件给予我学业支持的情况上，我还没有真正踏足社会，不知道残酷的竞争里究竟有没有我的容身之所，因而现在让我最害怕的，或许就是我强烈的理想化与对成长可能性的向往终将要被残酷的现实撕裂，最后让我狼狈地被驱逐回我现在唯恐避之不及的地方。那时候，或许就是社会要给我锤出血洞洞，我也要给我自己锤出伤口。（现在还悲观地想过，估计以后很难混得体面了，能独自一人生存下来似乎就很好，找到伴侣也会是奢望。毕竟，错过了大学，一来再难遇上纯真的爱情，二来即便不谈论美好的浪漫爱，基于现实考量，也不会有谁愿意和经济能力不足的我在一起。可能现在想到这些太远了，很离谱，但我觉得现在的社会风气就给我这样的感觉，而且大家似乎也这样想……）不过，我这些都是想，还没有真正遭遇，等我真的进社会了，也会慢慢和你一样被锤的，只是不知道我会感到有多疼。但我现在也已经开始时不时害怕。

我现在其实还会下意识地把许多人当作自己的“救命稻草”，然而荒谬的是，对于真正、最爱我的血亲们，我的父母、我的外公奶奶，我却更多愿意瞒着他们我的状况，不愿意和他们吐露近况，但对于一些陌生人，我却将不少情感投放在对方身上，现在想来当真觉得自己还有许多地方不够成熟。待人真诚是重要的，但不应当在毫无分辨的情况下形成对对方依赖和期待的情感。我到现在还做过这样的有点傻气的事情，前几天和一位陌生朋友出行的经历就让我想到这一点。就此而言，我觉得就可以呼应你起头说的，自己应当能做到填满自己的一部分，爱自己，爱世界。但这会是一个漫长的过程……

**Reply by earfanfan** [reply]
武汉市群众艺术馆这几年都会举办一个“百姓明星”大赛，今天下午一点到三点是曲艺类的决赛，昨天歌唱类和舞蹈类本来也是想去看的但是因为别的事耽搁了，今天我打算去看，不过骑车过去也得半个多小时，所以我先长话短叨叨一下吧。

你感到恐惧害怕的东西我也怕，哈哈哈，不过我想说怕就怕吧，怕是很正常的事情，等到你一步步亲身经历的时候，大概又会有许多新的体会，有些会和你现在害怕的不一样，也许也有一样的。如果害怕的话，就对自己说，老子真得超怕的啊。可是人生总是要往前走的，专注于脚下的路，如果你有自己的想法，就去尝试，非要用功利的角度去评判的话，我这过来人感觉年轻的时候试错的资本好多啊。

另外，所谓残酷现实是否将你撕裂这件事，其实决定权是在你自己手上的。我给你讲讲我的狗弟弟的故事，这个是我的亲弟弟，他小时候没有好好读书，现在做的是卖力气的活计，我让他去学点修车的技术，但他有孩子要养也拿不出钱来作出前期投入，我有时候看到益辉他们自嘲是搬砖的码农，我都会心酸一会，因为我弟弟才是真正生活在社会底层搬动真正的砖块、对生活毫无还手之力的人。可我这个狗弟弟活到现在，每天都接触底层社会太多残酷事实，他依然保持着他的本性，至少他选择继续做一个善良的人，不愿意为利益而坑害别人。

被戳出血洞洞来疼是疼，害怕是害怕，但是是选择积极自救还是轻易放弃自己又是另一回事。说不定以前被填进去的东西不是你自愿选择的，等到被戳穿以后，你又可以好好想想重新选择填点撒进去呢。

即便真的因为社会环境影响太大，不得不过上所谓“一眼望到头”的生活，你也还是可以多在生活本身上去想办法开拓一下的，比如发展下兴趣爱好或者循着自己的想法做点想做的事，生活的宽度呀、广度呀、深度呀或者随便什么度任君选择。

**Reply by residualsun1** [reply]
我回头再想想，觉得自己又是有些轻浮的。

某种程度来说，我的恐惧可能却是他人觉得是能满足的。昨晚，我的一位朋友和我说起他对自己的另一位朋友“只是向上看”，按照他的描述，他的那位朋友可能有一种“考不上研就很失败，觉得别人都看不起的心态”，因而接受不了其他可能性——或者说，看不到其他的可能性吧。我的这位朋友的看法是，向上看无可厚非，但应该在最有时间的年龄里多出去走走，看见自己身边的人，也看见自己“下面”的人，不是要说与他们比较，而是要认识到这个世界的参差，进而才会有足够的力量去接受不完美的自己，因为都看过见识过了，便也发现自己也是一个普通人，但“你”又还是比较幸运的，还能在大学里面“卷”，继续向上看。重要的是知道我是谁，找到自己的定位。

看到你分享你弟弟的故事时，我想起b站有一位up主，叫“摩的司机徐师傅”，他喜欢去寻找社会中的普通群众，将镜头对准他们，听他们自述故事。这些群众形形色色，有富二代、有普普通通的老百姓，也有被排斥的特殊群体，但大多数还是日常生活里安守本分的平凡人，偶尔看到他们的故事，会觉得人与人之间不同太多，不平等也随处可见。而过去，我总觉得自己这里不如别人，那里不如别人，但极少想过我已经拥有的，却又是另一些人一直所期盼拥有的（比如健康的身体、完整且能支持我完成学业的家庭、已然身处可自由学习的环境）。这样想的时候，我一面觉得自己“随意”同情别人是不够严肃的，因而我认可朋友说的在看到更多人时找到对自己的定位，这样才是生产力量的一种方式；另一面，在看到众生百态以后，在我看来，摆正自己的心态也是一种对他们的尊重。如果永远将自己放在一个既定的上升轨道去看——这无可厚非，但我认为从更大的角度来看，这种人对自己、他人乃至社会的认识都是割裂的、局限的、窄化的，不知不觉间心里只有自己，而自己也变成了标准的驱使，他们以后或许能过得很体面，很好地照顾自己和自己的“小家”，但就理想化来看，比如引用鲍曼关于社会的评判标准：“让最弱势的成员过上体面的生活”，我认为一个良善的社会需要的公民还需要做得更多，进一步说，国家的管理人员们更应该从形形色色的众生里走出来，再走进去。换言之，那些致力于追求上升的人阿，在理想的、道德伦理层面上来看，其实也担负着某些对他人和社会的责任，但至少就今天的许多人来说，只渴望上升的过程已经阻碍了这种责任形成的可能。

当然，这需要经济基础。对于需要以生存为主要任务的许多人来说，他们没有时间和精力去考虑太多，但这也从不妨碍他们保持善良的品性，而或许在大多数情况下，这部分人还都要更善良。所以我说的要看得更全面，主要适用于许多有一定条件，却仍然盲目地只看到一条上升轨道的人，轻易便觉得很绝望、没有希望的人，并非是要苛求他们必须认为自己对他人担负着什么责任，而至少不要觉得自己就是最惨的，总是郁郁不得志，而应该能看见更多人的处境，也主动认清自己的处境。我应该也是这类人，我的家庭没有糟糕到需要我去帮忙维系生存，当我想到我在进入社会以前得到的大量支持（举一些比较小的例子，我每月能拿2000的生活费，我想这已经比许多同龄人好上太多太多……），再看到那些还在底层挣扎的人（如同你说的，“**对生活毫无还手之力**”），我便觉得，那些理想化的追求和现实挑战之间的矛盾让我感受到的恐惧是轻浮的，或者更准确来说，我对此产生的“莫名的愤怒、恐惧和悲伤的交织”是轻浮的，是一种未经审视、不谙世事、自以为是、窄化和小家子气的自我认知产物——这让我难为情、羞愧。但现在更重要的是如何处理这种情绪。我想我所正在做的事情是一种回应，但我应该是对自己的定位想得还不够清楚，或者还不那么坚定，因此在准备的时间里一度产生各种各样的情绪。以上，我想我是以我的方式理解了你说的“所谓残酷现实是否将你撕裂这件事，其实决定权是在你自己手上的”。

再援引一段项飙过去在专访里说的一段话，有段时间，我感觉难过的时候都会看看，虽然总感觉自己的脑子都要成为他们观点的跑马场了哈哈：

> 今天的无力感是不是跟昨天的亢奋是一体两面的东西？如果社会给了我们一个正向的反馈，我们就很亢奋，觉得一切都是在向上的，觉得批判都是多余的；那么社会一旦给你的反馈是负面的，你一下子就会觉得是无能为力的，觉得勇敢是矫情的。这是没有看到社会和个人处在非常复杂的互动过程中，把生命过程想象成了实验室里老鼠和电击之间的那种反馈关系。
>
> 什么叫无力感的内在结构呢？就是说，无力感不是一个单一的感受，它背后有不同的因子。前面提到的预期是一个因子，**如果一个人的预期很单一，觉得我这么做一定要得到这样的结果，那就容易产生无力感。同时，无力感不仅仅是因为一个人受挫，而是说，你觉得你受挫的那件事情是唯一有价值的事情，这件事是要决定其他一切事情的。否则的话，当你失败的时候，你还可以愤怒、反思、另辟蹊径，无力感只不过是其中的一种可能**。

> 说不定以前被填进去的东西不是你自愿选择的，等到被戳穿以后，你又可以好好想想重新选择填点撒进去呢

这句很有深意，我觉得是要好好去思索的。最后，感谢你和我说的这些，从来不觉得你与我的交谈带有任何说教，而都是真诚的宽慰和建议: )。

**Reply by earfanfan** [reply]
>我回头再想想，觉得自己又是有些轻浮的。

好的，轻浮吧。

>或者更准确来说，我对此产生的“莫名的愤怒、恐惧和悲伤的交织”是轻浮的，是一种未经审视、不谙世事、自以为是、窄化和小家子气的自我认知产物——这让我难为情、羞愧。

好的，羞愧吧。

>虽然总感觉自己的脑子都要成为他们观点的跑马场。

好的，跑一匹别人的马吧。

>但我应该是对自己的定位想得还不够清楚，或者还不那么坚定

好的，不清楚吧，不坚定吧。

这些都没关系的。年轻的时候经历还少，人见得少，书也读的少，“少”是正常的，所以很容易就能认识到自己的“轻浮”，为某些事感到“羞愧”也都是正常的，这都是你认识自己的过程。悄咪咪地说，虽然你给自己很多负面评价，不过从我的视角看都是正面的印象，因为这都是你在迷雾中努力去分辨的过程，而且你真的很善于思考且从谏如流。从你的描述来看，你一直都被保护的很好啊，如果将来你进入社会被抽了几鞭子后疼得难受心中积攒了一些污浊的戾气，突然有一天很想要放下心中的真诚啊善良啊勇敢啊之类的特质的时候，或者说想要放下还犹豫不决的时候，你真的可以在那些时刻给我或者列弛写写信，然后再做决定也不迟。

不过，现在你还没毕业，思考和与人交流并不能让你不用找工作，所以担心找工作困难就多取取经，将来如果你要独立出去租房子住也可以多取取经，取经就是看看别人咋做的、都有哪些注意事项对你来说也实用，说不定以后能少踩点坑。
